examples/error-subtyping/error_subtyping.md

```diff
@@ -1,9 +1,10 @@
 # Error subtyping
 
-`distinct` creates a new subtype and can be used to define subtypes of `error`. The name of the distinct `error` type can be used with the error constructor to create an `error` value of that type.
-
-Works like a nominal type. The `is` operator can be used to distinguish distinct subtypes. Each occurrence of `distinct` has a unique identifier that is used to tag instances of the type.
+A given non-distinct `error` type `E1` is a supertype of another error type `E2` if and only if the detail record type of `E1` is a supertype of the detail record type of `E2`. If more explicit control over the error type relations is desired you can use `distinct` error types. Each declaration of a distinct error type has a unique type ID. If the supertype is a `distinct` error type then there is the additional requirement that it must contain all the type IDs of the subtype. 
```

Comment:
Wondering if it is better to give the explanation with the symbols used in the example itself rather than `E1` and `E2`. @heshanpadmasiri @MaryamZi WDYT?

Reply:
Fixed